Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -21,6 +21,8 @@
 SERVER = demisto.params().get('url', '').rstrip('/')
 USERNAME = demisto.params().get('credentials', {}).get('identifier')
 PASSWORD = demisto.params().get('credentials', {}).get('password')
+API_KEY = demisto.params().get('api_key')
```

Comment:
See my comment: https://github.com/demisto/etc/issues/22395

Reply:
I changed the field to Authorization field

Reply:
There should be one credential field. See: https://github.com/demisto/etc/issues/22395#issuecomment-594668009